### ISSUE TITLE
Underline does not show on initial load for customized Tab Bar (v0.6.7)

### ISSUE THREAD

**Issue by oureta** [happypancake/react-native-scrollable-tab-view#672]
```
<ScrollableTabView>
  renderTabBar={tabBarProps => <ScrollableTabBar {...tabBarProps} />}
...
```

**Comment by zhangxiaoyu1314** [comment]
+1

**Comment by GreyLove** [comment]
+1

**Comment by rvpyrv** [comment]
the same issue, but for default scrollableTabView.

**Comment by Clcll** [comment]
the same. .. did you solve this issue ?

**Comment by batoan1221** [comment]
does someone have any solution? same issue for <ScrollableTabBar />

**Comment by thaiminh** [comment]
same issue here :(

**Comment by nishiltamboli** [comment]
Downgraded to 0.6.6 and the first tab opens up fine.

**Comment by CodingMonkeyzh** [comment]
replace `this.props.scrollValue._value` to `this.props.scrollValue.__getValue()` can resolve this issue.

**Comment by oureta** [comment]
This also happens on v0.8.0. @skv-headless, i know you don't want to add `tabBarStyle` prop but with this issue it won't be possible to remove the bottom border.

**Comment by wanghz2** [comment]
possible fix in caller side? overwrite the imported function?

**Comment by arfaWong** [comment]
same issue here on  v0.8.0. @skv-headless.
I manually replace `this.props.scrollValue._value` to `this.props.scrollValue.__getValue()` and resolve this issue.

**Comment by gmaggio** [comment]
The suggested `this.props.scrollValue.__getValue()` seems to fix the problem. Just wondering but why isn't it implemented yet?

**Comment by wanghz2** [comment]
@skv-headless we are expecting you have plan checkin the fix into v0.8.next release, to avoid we re-package or find not ideal workaround from caller side.

**Comment by ptomasroos** [comment]
Fixed in master, will be released later today
